neo/rawio/intanrawio.py

```diff
@@ -591,7 +648,7 @@ def read_variable_header(f, header):
     ("electrode_impedance_phase", "float32"),
 ]
 
-stream_type_to_name_rhs = {
+stream_id_to_name_rhs = {
```

Comment:
Nice renaming!

Reply:
Maybe we can go further as there are other references to type instead of id in the code but I wanted to keep the diff small for the first review.

Reply:
OK, I went further and changed signal_type for stream_id in most places. Check it out.